packages/http-client-csharp/eng/pipeline/publish.yml

```diff
@@ -41,3 +41,4 @@ extends:
           StagePrefix: "CSharp"
           LanguageShortName: "csharp"
```

Comment:
Do we even need the short name now? I would expect every package here will need the fully qualified name

Reply:
Its used in other places like step naming etc, apiview, etc.

Reply:
wonder then if you should just use that instead of the coalesce